ca/config.go

```diff
@@ -425,36 +442,18 @@ func RenewTLSConfigNow(ctx context.Context, s *SecurityConfig, connBroker *conne
 		return err
 	}
 
-	clientTLSConfig, err := NewClientTLSConfig(tlsKeyPair, rootCA.Pool, CARole)
+	clientTLSConfig, err := NewClientTLSConfig([]tls.Certificate{*tlsKeyPair}, rootCA.Pool, ManagerRole)
```

Comment:
Both are fine, since the manager certificate has both as DNS names, but for consistency I changed this to the manager role instead.